sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_quickpulse/_live_metrics.py

```diff
@@ -29,9 +53,24 @@ def enable_live_metrics(**kwargs: Any) -> None:
     _QuickpulseManager(kwargs.get('connection_string'), kwargs.get('resource'))
 
 
+# Used by _QuickpulseSpanProcessor to record live metrics on span record
+def record_span(span: ReadableSpan) -> None:
```

Comment:
If these are only used by the processors, I think we should just have this logic be in the processors. This feels like an unnecessary intermidiary